graphql/resolvers.go

```diff
@@ -137,6 +137,23 @@ func (r *queryResolver) MyPublicKeys(ctx context.Context) ([]*restModel.APIPubKe
 	return publicKeys, nil
 }
 
+func (r *taskResolver) Project(ctx context.Context, obj *restModel.APITask) (*restModel.APIProjectRef, error) {
+	pRef, err := r.sc.FindProjectById(*obj.ProjectId)
```

Comment:
I just added a change that will break this; you'll want to set the new parameter to "true" in order to get the full project